### PR TITLE
grammalecte: init at v0.5.17

### PR DESCRIPTION
**Pull request by apeyroux**
###### Motivation for this change

Grammalecte is an open source grammar checker dedicated to the French language. Grammalecte est un correcteur grammatical open source dédié à la langue française

###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -7126,6 +7126,30 @@ in {
     };
   };
 
+  grammalecte = buildPythonPackage rec {
```
Comment (Mic92) [inline pkgs/top-level/python-packages.nix]: Can you move this python package out of this file as described in the header of this file?

Comment (Mic92) [inline pkgs/top-level/python-packages.nix]: This looks like an application and not a python library to me. If this is the case please move it out of `python-packages.nix` into its own file and reference it in `pkgs/top-level/all-packages.nix`

Comment (apeyroux) [inline pkgs/top-level/python-packages.nix]: ok

Comment (Mic92) [inline pkgs/top-level/python-packages.nix]: If it is a python library then you can keep it here. However the expression to `python-modules` as described in the header of this file.

**Comment by FRidh** [comment]
pushed https://github.com/NixOS/nixpkgs/commit/be2b6c2e473d843f56419e34834124baace46f0d

**Pull request closed without merging**